membership-common/src/main/scala/com/gu/memsub/subsv2/reads/SubJsonReads.scala

```diff
@@ -129,7 +129,7 @@ object SubJsonReads {
               (__ \ "ReaderType__c").readNullable[String].map(ReaderType.apply) and
               (__ \ "GifteeIdentityId__c").readNullable[String] and
               (__ \ "autoRenew").read[Boolean]
-          )(memsub.subsv2.Subscription.partial[P](hasPendingFreePlan) _).reads(o)
+          )(memsub.subsv2.Subscription.partial(hasPendingFreePlan) _).reads(o)
```

Comment:
What does hasPendingFreePlan mean now that we don't have free plans?

Reply:
good spot, I've done that in another PR that's going out this morning

Reply:
although feedback on whether I'm balancing up the size vs number of PRs well would be useful

Reply:
I think you've got it about right